gax/src/streamingCalls/streaming.ts

```diff
@@ -203,7 +544,12 @@ export class StreamProxy extends duplexify implements GRPCCallResult {
 
     const stream = apiCall(argument, this._callback) as CancellableStream;
     this.stream = stream;
-    this.forwardEvents(stream);
+
+    if (this.new_retry) {
```

Comment:
So to summarize the names comments above, let's find some good name instead of `new` (based on supported features, not by the timeline).

Reply:
Makes sense, done!